lib/gui/os/window-progress.js

```diff
@@ -38,12 +39,16 @@ exports.currentWindow = electron.remote.getCurrentWindow();
  * Show progress inline in operating system task bar
  *
  * @param {Number} percentage - percentage
+ * @param {String} status - flashing/validating status
  *
  * @example
  * windowProgress.set(85);
  */
-exports.set = (percentage) => {
+exports.set = (percentage, status) => {
```

Comment:
Hm, although that now raises the question of whether this functionality should live in this module at all. Maybe worth creating a new function that simply sets the window title, and nothing else?